### PR TITLE
Implement weeklyCaseDensity

### PR DESCRIPTION
**Pull request by igorkofman**
This PR addresses issue https://trello.com/c/mQAfsaWF/399-add-cdc-community-level-related-metrics-to-api

### Please Check
- [x] Are the [JSON schemas](https://github.com/covid-projections/covid-data-model/tree/main/api/schemas) updated?
- [x] Are tests passing?

### CODE REVIEW

File: tests/libs/api_v2_pipeline_test.py [hunk 1]
```diff
@@ -307,7 +307,7 @@ def test_annotation_all_fields_copied(rt_dataset):
     )
 
     timeseries_for_region = api_v2_pipeline.build_timeseries_for_region(regional_input)
-
+    print(timeseries_for_region.annotations)
```
Comment (mikelehen) [inline tests/libs/api_v2_pipeline_test.py:310]: remove?

**Review by mikelehen — CHANGES_REQUESTED** [review]
Looks reasonable, though I don't know this code very well so if there was something missing / wrong I probably wouldn't know.  Have you built an API snapshot or run the API generation locally?  Is there a generated API endpoint I could look at (feel free to throw it in a gist or whatever), ideally including timeseries data?

**Comment by igorkofman** [comment]
@mikelehen  https://data.covidactnow.org/snapshot/3100/v2/cbsa/10100.timeseries.json
https://data.covidactnow.org/snapshot/3100/v2/county/20033.timeseries.json
https://data.covidactnow.org/snapshot/3100/v2/county/51085.json
https://data.covidactnow.org/snapshot/3100/v2/state/AZ.json
https://data.covidactnow.org/snapshot/3100/v2/states.timeseries.json
https://github.com/covid-projections/covid-data-model/runs/5668362439?check_suite_focus=true

**Review by mikelehen — CHANGES_REQUESTED** [review]
Looks like the metric may not be showing up in CSVs for some reason. https://data.covidactnow.org/snapshot/3100/v2/states.csv

Maybe it needs to be added to https://github.com/covid-projections/covid-data-model/blob/main/libs/pipelines/csv_column_ordering.py ?

**Review by mikelehen — APPROVED** [review]
Changes look good assuming this successfully gets it into the CSVs! 🤞

**Pull request merged**